<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterFace</class>
 <widget class="QWidget" name="InterFace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>658</width>
    <height>365</height>
   </rect>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="UserName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLabel" name="Credit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>150</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>credit</string>
   </property>
  </widget>
  <widget class="QLabel" name="SystemTime">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>time</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="Refresh">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="commandLinkButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>LogOut</string>
   </property>
  </widget>
  <widget class="QLabel" name="DriverName">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>DriverName</string>
   </property>
  </widget>
  <widget class="QLabel" name="TripStatus">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>TripStatus</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cost">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>80</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>Cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="Distance">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>40</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>Distance</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LastTrip">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>40</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Current Trip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ChargeBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>90</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Charge</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RateBtn">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>280</y>
     <width>101</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Rate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RequestBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>190</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Request</string>
   </property>
  </widget>
  <widget class="QPushButton" name="EstimateBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>240</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Estimate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CancelTrip">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>160</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Cancel Trip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CodeBtn">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>140</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Discount Code</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>110</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Discount Code :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Code">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>CODE21</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>190</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Number  :</string>
   </property>
  </widget>
  <widget class="QLabel" name="CodeNum">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QLabel" name="Model">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>200</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>model</string>
   </property>
  </widget>
  <widget class="QLabel" name="Color">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>240</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'red'</string>
   </property>
   <property name="text">
    <string>color</string>
   </property>
  </widget>
  <widget class="QLabel" name="Distance_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Distance :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cost_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Cost :</string>
   </property>
  </widget>
  <widget class="QLabel" name="TripStatus_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>TripStatus :</string>
   </property>
  </widget>
  <widget class="QLabel" name="DriverName_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>160</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Model_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>model :</string>
   </property>
  </widget>
  <widget class="QLabel" name="Color_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>240</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>color :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>661</width>
     <height>371</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>UserName</zorder>
  <zorder>Credit</zorder>
  <zorder>SystemTime</zorder>
  <zorder>Refresh</zorder>
  <zorder>commandLinkButton</zorder>
  <zorder>DriverName</zorder>
  <zorder>TripStatus</zorder>
  <zorder>Cost</zorder>
  <zorder>Distance</zorder>
  <zorder>LastTrip</zorder>
  <zorder>ChargeBtn</zorder>
  <zorder>RateBtn</zorder>
  <zorder>RequestBtn</zorder>
  <zorder>EstimateBtn</zorder>
  <zorder>CancelTrip</zorder>
  <zorder>CodeBtn</zorder>
  <zorder>label</zorder>
  <zorder>Code</zorder>
  <zorder>label_2</zorder>
  <zorder>CodeNum</zorder>
  <zorder>Model</zorder>
  <zorder>Color</zorder>
  <zorder>Distance_2</zorder>
  <zorder>Cost_2</zorder>
  <zorder>TripStatus_2</zorder>
  <zorder>DriverName_2</zorder>
  <zorder>Model_2</zorder>
  <zorder>Color_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
